<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>30</y>
      <width>188</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Список фильтров</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="rgbtolumAct">
       <property name="text">
        <string>Обесцвечивание</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gaussianfilterAct">
       <property name="text">
        <string>Размытие</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hrgradientAct">
       <property name="text">
        <string>Горизонтальный градиент</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="vtgradientAct">
       <property name="text">
        <string>Вертикальный градиент</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="borderdetectAct">
       <property name="text">
        <string>Выделение контура</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fenetragefilterAct">
       <property name="text">
        <string>Гамма</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="seuillagefilterAct">
       <property name="text">
        <string>Пороговый фильтр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="binarisationfilterAct">
       <property name="text">
        <string>Бинаризация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="seuilageIntervallefilterAct">
       <property name="text">
        <string>Усреднение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="binaristionIntervallefilterAct">
       <property name="text">
        <string>Медианный фильтр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inversionfilterAct">
       <property name="text">
        <string>Инверсия</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Image</string>
     </attribute>
     <widget class="QLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>941</width>
        <height>581</height>
       </rect>
      </property>
      <property name="text">
       <string>Для того, чтобы обработать изображение, нажмите &quot;Файл&quot;,
 затем &quot;Открыть&quot; и выберите изображение</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QSlider" name="slideA">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>670</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slideB">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>700</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>A:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LcdA">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>670</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LcdB">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>700</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="initialiserAct">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>560</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Исходное состояние</string>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>groupBox</zorder>
   <zorder>slideA</zorder>
   <zorder>slideB</zorder>
   <zorder>labelA</zorder>
   <zorder>labelB</zorder>
   <zorder>LcdA</zorder>
   <zorder>LcdB</zorder>
   <zorder>initialiserAct</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="ouvrirAct"/>
    <addaction name="separator"/>
    <addaction name="enregistrerAct"/>
    <addaction name="separator"/>
    <addaction name="fermerAct"/>
   </widget>
   <widget class="QMenu" name="menuApropos">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutAct"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuApropos"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="ouvrirAct">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="enregistrerAct">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="fermerAct">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="aboutAct">
   <property name="text">
    <string>Приложение</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>slideA</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LcdA</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slideB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LcdB</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>ouvrir()</signal>
  <signal>fermer()</signal>
  <signal>enregistrer()</signal>
  <signal>about()</signal>
  <signal>inversion()</signal>
  <signal>rgb2luminance()</signal>
  <signal>gaussianfilter()</signal>
  <signal>fenetragefilter()</signal>
  <signal>seuillagefilter()</signal>
  <signal>binarisationfilter()</signal>
  <signal>seuilageintervallefilter()</signal>
  <signal>binarisationintervallefilter()</signal>
  <signal>initialiser()</signal>
  <signal>hrgradient()</signal>
  <signal>vrgradient()</signal>
  <signal>border_detect()</signal>
 </slots>
</ui>
